.NET/Microsoft.Recognizers.Text.Sequence/SequenceRecognizer.cs

```diff
@@ -9,10 +11,22 @@ public class SequenceRecognizer : Recognizer
 
         private SequenceRecognizer(SequenceOptions options)
         {
-            RegisterModel(Culture.English, options.ToString(), new Dictionary<Type, IModel>
+            var models = new Dictionary<Type, IModel>
```

Comment:
Wouldn't a better approach be to just override the ways to get a model to ignore culture?